test/unit/elsa/group/acknowledger_test.exs

```diff
@@ -1,8 +1,8 @@
 defmodule Elsa.Group.AcknowledgerTest do
   use ExUnit.Case
 
-  alias Elsa.{Registry, Supervisor}
   alias Elsa.Group.Acknowledger
+  alias Elsa.{Registry, Supervisor}
```

Comment:
nit, but this format makes it really annoying to search for aliases when doing renames.